xapian-core/include/xapian/weight.h

```diff
@@ -435,6 +449,7 @@ class XAPIAN_VISIBILITY_DEFAULT TfIdfWeight : public Weight {
      *     @li 'b': Boolean    wdfn=1 if term in document else wdfn=0
      *     @li 's': Square     wdfn=wdf*wdf
      *     @li 'l': Logarithmic wdfn=1+log<sub>e</sub>(wdf)
+     *     @li 'm': Max-wdf     wdfn=wdf/max-wdf
```

Comment:
`max-wdf` in the formula could be mis-read as subtraction - `max_wdf` would avoid that.